articles/connections/database/custom-db/templates/login.md

```diff
@@ -440,28 +431,22 @@ function login(email, password, callback) {
 
   connection.connect();
 
-  var query = "SELECT id, nickname, email, password " +
-    "FROM users WHERE email = ?";
+  const query = 'SELECT id, nickname, email, password FROM users WHERE email = ?';
```

Comment:
In Heroku review, code block renders strangely (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/login#mysql